enums.go

```diff
@@ -0,0 +1,282 @@
+package openrtb
+
+//enums and constants for tables 5.x.x (except 5.1 - IAB categories)
```

Comment:
Check Go std lib - leaving space after `//` is more common.

So, `// comment` is cool, but `//comment` is not that cool.